tests/test_rigid_physics.py

```diff
@@ -1100,3 +1101,56 @@ def test_equality_weld(gs_sim, mj_sim):
     qpos = gs_sim.rigid_solver.dofs_state.pos.to_numpy()[:, 0]
     qpos[0], qpos[1], qpos[2] = 0.1, 0.1, 0.1
     simulate_and_check_mujoco_consistency(gs_sim, mj_sim, qpos, qvel, num_steps=300, atol=atol)
+
+
+@pytest.mark.parametrize("backend", [gs.cpu])
+def test_mesh_to_heightfield(show_viewer):
+
+    ########################## create a scene ##########################
+    scene = gs.Scene(
+        show_viewer=show_viewer,
```

Comment:
for consistency with all the other tests, keep the various options in the same order:
* sim -> rigid -> viewer -> show_*